go.mod

```diff
@@ -1,58 +1,36 @@
 module github.com/grafana/xk6-output-prometheus-remote
 
-go 1.17
+go 1.18
 
 require (
-	github.com/golang/protobuf v1.5.2
+	github.com/gogo/protobuf v1.3.2
```

Comment:
The project is deprecated. There is even a youtube video around it https://www.youtube.com/watch?v=HTIltI0NuNg

Not sure if we need it to do in this PR, but at some point, we should migrate out of it :sweat_smile:

Reply:
If it's not a big deal, maybe we can revert this change in this PR not to produce new tasks.